derive_builder/tests/builder_field_custom.rs

```diff
@@ -0,0 +1,51 @@
+#[macro_use]
+extern crate pretty_assertions;
+#[macro_use]
+extern crate derive_builder;
+
+use std::num::ParseIntError;
+
+#[derive(Debug, PartialEq, Default, Builder, Clone)]
+pub struct Lorem {
+    #[builder(field(type = "Option<usize>", build = "self.ipsum.unwrap_or(42) + 1"))]
+    ipsum: usize,
+
+    #[builder(field(type = "String", build = "self.dolor.parse()?"))]
+    dolor: u32,
+}
+
+impl From<ParseIntError> for LoremBuilderError {
+    fn from(e: ParseIntError) -> LoremBuilderError {
+        LoremBuilderError::ValidationError(e.to_string())
+    }
+}
+
+#[test]
+fn custom_fields() {
+    let x = LoremBuilder::default().dolor("7".into()).build().unwrap();
```

Comment:
Can you drop the `into` here, and instead make this an `into` setter above? I think it'd be good to show that feature working in tandem with custom field types.